tests/zfs-tests/tests/functional/cli_root/zfs_copies/zfs_copies_005_neg.ksh

```diff
@@ -43,11 +43,6 @@
 
 verify_runnable "global"
 
-# See issue: https://github.com/zfsonlinux/zfs/issues/6145
-if is_linux; then
-	log_unsupported "Test case occasionally fails"
-fi
```

Comment:
I tried enabling some SKIPped tests and this wasn't failing; i just discovered the cleanup issue with a custom runfile.
The reason this was disabled is in 95401cb6f7aa9ec3dd2d61335d2dcc9459c2661c: _Skip until layering pools on zvols is solid_. Here $TESTDIR is a ZFS mountpoint, i could move this to $TEST_BASE_DIR like i did with the other file VDEVs (i'll also update the commit message with the next push).

Reply:
That would explain it.  Layering pools on zvols is much better than it used to be but still doesn't seem to be 100% solid and could deadlock.  Since we don't need to layer a pool on a zvol to perform this test I think moving it as you suggest would be best.  I'd feel much better about re-enabling it then.